ADFSToolkit/Public/Update-ADFSTkInstitutionConfiguration.ps1

```diff
@@ -43,12 +43,18 @@
         #Check if the federation dir exists and if not, create it
         ADFSTk-TestAndCreateDir -Path $defaultFederationConfigDir -PathName "$federationName config directory"
 
-        Write-ADFSTkHost -WriteLine -AddSpaceAfter
-        Write-ADFSTkHost confCopyFederationDefaultFolderMessage -Style Info -AddSpaceAfter -f $defaultFederationConfigDir
+        #Check if we already have any Federation defaults file(s)
+        $allDefaultFederationConfigFiles = Get-ChildItem -Path $defaultFederationConfigDir -Filter "*_defaultConfigFile.xml"
```

Comment:
I notice that this looks in the root folder of the Config dir but no deeper. Is that intended (I think so).
Further on down there's a -Recurse directive on the fetch  -- should this and the other item at line 56 be identical?

Reply:
The $defaultFederationConfigDir id built up from the Federation Dir and the Federation Name:
$defaultFederationConfigDir = Join-Path $Global:ADFSTkPaths.federationDir $federationName
so the -Recurse param shouldn't be needed. I don't know why I put it in the lower section.